modules/apps/depot/depot-service/src/main/java/com/liferay/depot/internal/model/listener/GroupModelListener.java

```diff
@@ -110,6 +125,48 @@ private void _copyDepotAppCustomizations(
 		}
 	}
 
+	private void _copyLiveDepotEntryGroupRelsToStage(Group group)
+		throws PortalException {
+
+		Group liveGroup = group.getLiveGroup();
```

Comment:
Even though you're guarding methods calls so that it is only called when staged, can you add a null check here? (liveGroup will be null if not staged).

Reply:
pushed on [fbb3daf](https://github.com/liferay-lima/liferay-portal/pull/3504/commits/fbb3daf686b9c27da50fa2210b3cd892639acad6)